### ISSUE TITLE
Recipes missing

### ISSUE THREAD

**Issue by blockbreaker1361** [TheCBProject/EnderStorage#3]
The recipe for the ender-pouch works fine, bit the recipes for the chest and tank will not come up for me. I am using the latest version of Forge and JEI.. I did not get a crash, so no log.

**Comment by ArMoReDfUrY** [comment]
the recipes do work they just don't show up in JEI.
the chest recipe is:
-blaze rods in the corners
-vanilla chest - center
-ender pearl - bottom center
-wool - top center
-obsidian blocks - left & right of the chest.
For the tank swap out the Vanilla Chest with a cauldron.

hope this helps for now :)

**Comment by blockbreaker1361** [comment]
Yeah. I tested in creative and they are craftable. He just needs to add or update JEI integration.

**Comment by gigabit101** [comment]
I assume they are using a custom crafting handler to allow colouring in crafting

**Comment by gigabit101** [comment]
This will need a JEI handler to make it show in JEI

**Comment by covers1624** [comment]
At the moment only the basic recipes are available, so the default ones with colored wool.
I need to rewrite the recipe handler for the re-coloring as the items color is stored on NBT now not metadata.

**Comment by b0bst3r** [comment]
And the diamond nugget to lock it privately?

**Comment by covers1624** [comment]
Just a normal diamond, it has a proper config value now.
